calculator.rb

```diff
@@ -0,0 +1,83 @@
+
+puts "*** Welcome to Natalie's Calculator! ***
+1. add(+)
+2. subtract(-)
+3. multiply(*)
+4. divide(/)
+-----------------
+Please chose an operator (name or symbol):"
+operator = gets.chomp
+
+# an array with all possible operators
+operators = ["+", "add", "-", "subtract", "*", "multiply", "times", "/", "divide"]
+
+until operators.include?(operator)
+  puts "That's not a valid operator. Select from the following list:
+  1. add(+)
+  2. subtract(-)
+  3. multiply(*)
+  4. divide(/)"
+  operator = gets.chomp
+end
+
+# The program handles erroneous input. For example the user might enter clown when asked to enter a number
+def is_string(a)
+  return a.to_f == 0 && a != "0" 
+end
+
+# this is getting set to cloud or whatever string 
+puts "Give me the first number to #{ operator }"
+first_number = gets.chomp
+while is_string(first_number)
+  puts "that's a string, pls give me a number"
+  first_number = gets.chomp
+end
+
+puts "Give me the second number to #{ operator }"
+second_number = gets.chomp
+while is_string(second_number)
+  puts "that's a string, pls give me a number"
+  second_number = gets.chomp
+end
+
+def add(a,b)
```

Comment:
You might consider using puts for the string and returning the answer (sum, difference, etc).